rpcs/ledgerService.go

```diff
@@ -117,21 +117,21 @@ func (ls *LedgerService) ServeHTTP(response http.ResponseWriter, request *http.R
 	genesisID, hasGenesisID := pathVars["genesisID"]
 	if hasVersionStr {
 		if versionStr != "1" {
-			logging.Base().Debugf("http ledger bad version '%s'", versionStr)
+			logging.Base().Debugf("LedgerService.ServeHTTP: bad version '%s'", versionStr)
```

Comment:
fine change, just making sure it's not related to the PR?
edit: oh i see it's in the list of changes